stats/informationbox.cpp

```diff
@@ -0,0 +1,72 @@
+#include "informationbox.h"
+#include "statscolors.h"
+#include <QChart>
+#include <QFontMetrics>
+
+static const QColor informationBorderColor(Qt::black);
```

Comment:
we need to remember to then do a pass through all this and make this dark mode aware :-/